Kconfig

```diff
@@ -100,6 +100,13 @@ config SID_HAL_PROTOCOL_MEMORY_SZ
 	int
 	default 1024
 
+config SIDEWALK_APPLICATION_NAME
+	string "Name of the application printed on start up"
+	default ""
```

Comment:
consider setting the defautl name linke "custom_app" or sth.  The "APP NAME" string will pirint the name, even when it's empty.

Reply:
ok

Reply:
set default to "sidewalk"